packages/suite/src/utils/wallet/accountUtils.ts

```diff
@@ -1,8 +1,35 @@
-import { Account } from '@suite-common/wallet-types';
+import { FiatCurrencyCode } from '@suite-common/suite-config/src/fiat';
+import { Account, RatesByKey } from '@suite-common/wallet-types';
 import { StaticSessionId } from '@trezor/connect';
+import { getTotalFiatBalance } from '@suite-common/wallet-utils/src/accountUtils';
 
+import { useSelector } from 'src/hooks/suite';
 import { WalletParams } from 'src/types/wallet';
 
+import { getTokens } from './tokenUtils';
+
+export const useTotalFiatBalance = (
```

Comment:
I think this should be reusable also with suite native, please move it into some package in `suite-common`

Reply:
done (moved it to hooks for now)